pkgs/by-name/re/refine/package.nix

```
@@ -0,0 +1,66 @@
+{
+  lib,
+  fetchFromGitLab,
+  nix-update-script,
+  blueprint-compiler,
+  desktop-file-utils,
+  libadwaita,
+  meson,
+  ninja,
+  pkg-config,
+  wrapGAppsHook4,
+  python3,
+  libxml2,
+  python3Packages,
+  libportal,
+  libportal-gtk4,
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "refine";
+  version = "0.2.0";
+  pyproject = false; # uses meson
+
+  src = fetchFromGitLab {
+    domain = "gitlab.gnome.org";
+    owner = "TheEvilSkeleton";
+    repo = "Refine";
+    tag = version;
+    hash = "sha256-NmfsogXkJicZj/cE9HFz91ERD9fxOGKoBV4h68KIE+g=";
+  };
+
```

```suggestion

  strictDeps = true;

```
Since we're dealing with native and potentially cross versions of packages at once